### ISSUE TITLE
receive a error message with "swift_version" when run "bundle exec pod install"

### ISSUE THREAD

**Issue by rainedAllNight** [grab/cocoapods-pod-merge#7]
My error messgae:
`
The `Networking` pod failed to validate due to 2 errors:
    - WARN  | github_sources: Github repositories should end in `.git`.
    - ERROR | [iOS] attributes: Unacceptable type `Array` for `swift_version`. Allowed values: `[String]`.
    - ERROR | [iOS] attributes: Unable to validate `swift_version` (Malformed version number string [5.0]).

`

My MergeFile:
`
group 'Networking'
	pod 'SwiftyJSON', '~> 5.0.0'
	pod 'ObjectMapper', '~> 3.5.0'
end

`

**Comment by biocross** [comment]
Hello @rainedAllNight,

The MergeFile you mentioned works for me: 
```
group 'Networking'
pod 'SwiftyJSON', '5.0.0'
pod 'ObjectMapper', '3.5.0'
end
```

I see your error is related to validation. Is there a specific reason you're trying to `validate` the merged pod?

**Comment by biocross** [comment]
Also, if you want the merged podspec to validate, you can try `bundle exec pod lib lint  --allow-warnings`. `pod spec lint` will not work unless you upload the merged pod onto a repository of it's own and tag it.

**Comment by rainedAllNight** [comment]
i have fixed it with update cocoapods version to 1.8.1
